domain/domain.go

```diff
@@ -254,6 +254,9 @@ func (do *Domain) tryLoadSchemaDiffs(m *meta.Meta, usedVersion, newVersion int64
 		if err != nil {
 			return false, nil, err
 		}
+		if diff.Type == model.ActionUpdateTiFlashReplicaStatus || diff.Type == model.ActionSetTiFlashReplica {
```

Comment:
After calling `tryLoadSchemaDiffs`, we will call `schemaValidator.Update` to affect `latestSchemaVer`.  So this  `latestSchemaVer` will still be added to the deltaSchemaInfos queue. I don't think this change can get the optimization you want.